buildbot/configure.py

```diff
@@ -199,6 +211,8 @@ def main():
     parser.add_argument("--use-lld", action="store_true", help="Use LLD linker for build")
     parser.add_argument("--llvm-external-projects", help="Add external projects to build. Add as comma seperated list.")
     parser.add_argument("--ci-defaults", action="store_true", help="Enable default CI parameters")
+    parser.add_argument("--ccache-dir", help="Enable CCache and store cache to specified directory.")
+    parser.add_argument("--ccache-max-size", help="Enable CCache and specify max cache size.")
```

Comment:
Adding ccache support looks unintuitive for me for two reasons:
1. There is `--cmake-opt` which can used to enable and configure ccache support.
2. This interface requires users to set either cache location or cache size just to enable ccache, but it should not be necessary requirement if I'd like to keep default values.

Overall, I don't see any value in adding these options `--cmake-opt` already does the job. The only added value is the check that `ccache` executable is available, but isn't it already checked by CMake?

Reply:
We're probably doing setup differently. I use LLVM_CCACHE_BUILD and other LLVM_ variables, which only enables ccache if it is available. You must be using CMAKE_CXX_COMPILER_LAUNCHER, which fails, if ccache is not found, and env variables.

Anyway, `--cmake-opt` didn't work for me, as it does not support multiple arguments, but I can probably fix it.